server/main.lua

```diff
@@ -42,6 +42,18 @@ end
 
 exports('ReleasePlayer', releasePlayer)
 
+RegisterNetEvent("qbx_prison:server:releasePlayer", function ()
+    local src = source
+    local Player = exports.qbx_core:GetPlayer(src)
+    if not Player then return end
```

Comment:
The only use case where a player would not exist is either the player is in character selection, or there is a bug with the GetPlayer export. If there is a bug, I don't think we'd want to silently fail, but should rather surface the error?